core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -179,7 +193,7 @@ public MultiMatchQueryBuilder field(String field, float boost) {
     /**
      * Sets the type of the text query.
      */
-    public MultiMatchQueryBuilder type(MultiMatchQueryBuilder.Type type) {
+    public MultiMatchQueryBuilder type(Type type) {
```

Comment:
Wonder if it makes sense to rename the inner `Type` here to something more specific to not get it confused with the three `MatchQuery.Type`s that it's using internally.